src/components/templates/loginForm/loginForm.tsx

```diff
@@ -1,70 +1,98 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import InputText from '../../atoms/inputs'
 import ButtonComponent from '../../atoms/button'
-import {Form} from 'semantic-ui-react'
-import { action } from '@storybook/addon-actions';
-import { StyledFormField } from './style/loginForm.styled.component';
+// import {Form} from 'semantic-ui-react'
+import {
+    Form,
+    SubmitButton,
+    Input
+  } from "formik-semantic-ui-react";
+import { Formik } from "formik";
+import * as Yup from "yup";
 
 
-export type loginFormInterface = {
+export type IloginForm = {
     isDisabled:boolean,
     isPrimary:boolean,
-    isLoading:boolean,
-    isError?:boolean,
+    isDisabledLogBtn?:boolean
 }
 
 const LogInForm = (
     {
         isDisabled = false,
         isPrimary = true,
-        isLoading = false,
-        isError = false,
-    }:loginFormInterface) => {
+    }) => {
+    
+    useEffect(() => {
+        setDisabledState(isDisabled);
+        setIsPrimaryState(isPrimary)
```

Comment:
I think I may be misunderstanding these lines of code. How is this different from just using the props directly?